lib/ex_doc/translations.ex

```diff
@@ -0,0 +1,304 @@
+defmodule ExDoc.Translations do
+
+  @locale_spanish_international "es"
+  @locale_spanish_spain "es-ES"
+  @locale_galician_spain "gl-ES"
+
+  def translate(@locale_spanish_international, "API Reference"), do: "Referencia de la API"
+  def translate(@locale_spanish_international, "Built using"), do: "Construido usando"
+  def translate(@locale_spanish_international, "Cancel search"), do: "Cancelar búsqueda"
+  def translate(@locale_spanish_international, "Collapse/expand sidebar"), do: "Contraer/expandir barra lateral"
+  def translate(@locale_spanish_international, "Cover"), do: "Cubierta"
+  def translate(@locale_spanish_international, "Display keyboard shortcuts"), do: "Mostrar combinaciones de teclas"
+  def translate(@locale_spanish_international, "Example"), do: "Ejemplo"
+  def translate(@locale_spanish_international, "Examples"), do: "Ejemplos"
+  def translate(@locale_spanish_international, "Exceptions"), do: "Excepciones"
+  def translate(@locale_spanish_international, "Function"), do: "Función"
+  def translate(@locale_spanish_international, "Functions"), do: "Funciones"
+  def translate(@locale_spanish_international, "Link"), do: "Enlace"
+  def translate(@locale_spanish_international, "Link to this "), do: "Enlace a esto "
```

Comment:
There's a trailing space
```suggestion
  def translate(@locale_spanish_international, "Link to this"), do: "Enlace a esto"
```

Reply:
Raw string: "Enlace a esto "